lib/qualys/element.rb

```diff
@@ -81,5 +87,21 @@ def method_missing(method, *args)
         return nil
       end
     end
+
+    private
+      
+    def cleanup_html(source)
+      result = source.dup
+      result.gsub!(/<P>/, "\n\n")
```

Comment:
@etdsoft only <P> tags (no `<p>` tags or `</P>` tags) are present in the collection of sample files that I have. But, I made them case-insensitive just in case. Also pushing a few more tags as I uncovered a few more tags while testing this.